addons/kubewatch.tf

```diff
@@ -0,0 +1,43 @@
+# BITNAMI KUBEWATCH CONTAINER HELM CHARTS
+# =======================================
+
+variable "kubewatch_enabled" {
+  description = "Kubewatch flag when enabled shall alert Slack about helm activities"
+  type        = bool
+  default = false
+}
+
+resource "kubernetes_namespace" "monitoring" {
+  count = var.kubewatch_enabled ? 1 : 0
+
+  metadata {
+    name = "monitoring"
+
+    labels = {
+      managed-by = "Terraform"
+    }
+  }
+}
+
+# Create the kubewatch operator, configure grafana dashboard ingress
+resource "helm_release" "kubewatch_operator" {
+  count      = var.kubewatch_enabled ? 1 : 0
+  name       = "kubewatch"
+  repository = "stable"
+  chart      = "kubewatch"
+  namespace  = "monitoring"
+
+  # Cleanup
+  provisioner "local-exec" {
+    when    = destroy
+    command = "helm delete --purge kubewatch"
+  }
```

Comment:
This isn't required, the helm_release resource should clean up after itself

Reply:
Removed `local-exec`